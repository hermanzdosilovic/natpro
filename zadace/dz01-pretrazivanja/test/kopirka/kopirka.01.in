8 6
20 62 19 89 19 50 25 66
