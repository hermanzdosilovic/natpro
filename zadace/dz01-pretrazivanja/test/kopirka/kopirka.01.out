20 / 62 19 / 89 / 19 / 50 25 / 66
